tests/E2E/SyncTest.php

```diff
@@ -12,31 +12,36 @@
 class SyncTest extends \PHPUnit_Framework_TestCase
 {
     /**
-     * @var Client
-     */
-    private $client;
-
-    public function setUp()
-    {
-        $this->client = new Client('b4c0n73n7fu1', 'cfexampleapi');
-    }
-
-    /**
      * @vcr e2e_sync_basic.json
      */
     public function testBasicSync()
     {
-        $mananger = $this->client->getSynchronizationManager();
+        $manager = (new Client('b4c0n73n7fu1', 'cfexampleapi'))
+            ->getSynchronizationManager();
 
-        $result = $mananger->startSync();
+        $result = $manager->startSync();
 
         $this->assertInstanceOf(Result::class, $result);
         $this->assertEquals('w5ZGw6JFwqZmVcKsE8Kow4grw45QdybCnV_Cg8OASMKpwo1UY8K8bsKFwqJrw7DDhcKnM2RDOVbDt1E-wo7CnDjChMKKGsK1wrzCrBzCqMOpZAwOOcOvCcOAwqHDv0XCiMKaOcOxZA8BJUzDr8K-wo1lNx7DnHE', $result->getToken());
         $this->assertTrue($result->isDone());
 
-        $result2 = $mananger->continueSync($result);
+        $result2 = $manager->continueSync($result);
 
         $this->assertInstanceOf(Result::class, $result2);
         $this->assertTrue($result2->isDone());
     }
+
+    /**
+     * @vcr e2e_sync_preview.json
+     */
+    public function testPreviewSync()
+    {
+        $manager = (new Client('e5e8d4c5c122cf28fc1af3ff77d28bef78a3952957f15067bbc29f2f0dde0b50', 'cfexampleapi', true))
+            ->getSynchronizationManager();
+
+        $result = $manager->startSync();
+
+        $this->assertInstanceOf(Result::class, $result);
+        $this->assertTrue($result->isDone());
```

Comment:
I think this is ok for now. I'll add some more exhaustive tests at some point in the future.